src/irmin-pack/unix/store_intf.ml

```diff
@@ -279,3 +309,29 @@ module type Maker_persistent =
     with type ('h, _) contents_key = 'h Pack_key.t
      and type 'h node_key = 'h Pack_key.t
      and type 'h commit_key = 'h Pack_key.t
+
+module type KV = sig
+  type endpoint = unit
+  type hash = Irmin.Schema.default_hash
+
+  include Pack_key.Store_spec
+
+  type metadata = Irmin.Metadata.None.t
+
+  module Make (C : Irmin.Contents.S) :
+    S
+      with module Schema.Contents = C
+       and type Schema.Metadata.t = metadata
+       and type Backend.Remote.endpoint = endpoint
+       and type Schema.Hash.t = hash
+       and type contents_key = (hash, C.t) contents_key
+       and type node_key = hash node_key
+       and type commit_key = hash commit_key
+       and type Schema.Path.step = string
+       and type Schema.Path.t = string list
+       and type Schema.Branch.t = string
+end
+
+(* This *_intf module does not have a Sigs module type as it is
+   directly exposed in the top-level interface
+   (irmin_pack_Unix.mli). *)
```

Comment:
Let's follow the pattern and add a `store.mli` that uses the the `Sigs` signature (and probably adds a `Maker` module). Then `irmin_pack_unix.mli` can use `Store.S`, etc.